app/views/editions/secondary_nav_tabs/send_to_fact_check_page.html.erb

```diff
@@ -0,0 +1,44 @@
+<% @edition = @resource %>
+<% content_for :title_context, @edition.title %>
+<% content_for :page_title, "Send to fact check" %>
+<% content_for :title, "Send to fact check" %>
+<% template = render template: "event_mailer/request_fact_check", formats: [:text] %>
+
+<div class="govuk-grid-row">
+  <div class="govuk-grid-column-two-thirds">
+
+    <%= form_for(:edition, url: send_to_fact_check_edition_path(@edition)) do %>
+      <%= render "govuk_publishing_components/components/input", {
+        label: {
+          heading_size: "m",
+          text: "Email addresses",
+        },
+        name: "email_addresses",
+        id: "email_addresses",
+        value: params[:email_addresses],
+        type: "email",
+        hint: "You can enter multiple email addresses if you comma separate them as follows: fact-checker-one@example.com, fact-checker-two@example.com",
+        input_html: { class: "form-control", required: true },
```

Comment:
Not necessarily saying this is wrong but does this do anything (here and below)?